Resources/Prototypes/Entities/Mobs/NPCs/space.yml

```diff
@@ -110,6 +110,8 @@
     interactFailureString: petting-failure-bear
     interactSuccessSound:
       path: /Audio/Animals/sloth_squeak.ogg
+  - type: CombatPower
+    factor: 2 # times more dangerous than a crew-member
```

Comment:
Magic values like this are immediately going to bitrot. If someone changes any aspect of a mob they shouldn't have to also be updating game director values.